guide/tutorial-4-more-about-controllers-20-minutes.md

```diff
@@ -128,12 +128,10 @@ Write the prototype of the `main()` function as follows:
 >   // initialize the Webots API
 >   wb_robot_init();
 >   // initialize devices
->   // feedback loop
->   while (1) {
+>   // feedback loop, continue until receiving an exit event
```

Comment:
The changes to the controllers code have to be ported to the webots repository as well.
In this case to the file https://github.com/omichel/webots/blob/master/projects/samples/tutorials/controllers/e-puck_avoid_collision/e-puck_avoid_collision.c#L32